api/src/main/java/org/openmrs/module/hospitalcore/InventoryCommonService.java

```diff
@@ -59,7 +59,9 @@ public interface InventoryCommonService extends OpenmrsService{
 
     public void voidRegimen(Regimen regimen) throws APIException;
 
-	List<Regimen> getRegimens (boolean voided);
+    public void createCycle(Cycle cycle) throws  APIException;
```

Comment:
we still need the getRegimens method that returns a list of Regimen objects. Noticed this has been deleted

Reply:
there was an issue with this commit lemme rectify this